components/canvas/webgl_thread.rs

```diff
@@ -830,6 +831,25 @@ impl WebGLThread {
                 "... rebound framebuffer {}, new back buffer surface is {:?}",
                 framebuffer_object, id
             );
+
+            // Tell Webrender to invalidate the caches for this image
```

Comment:
Sorry for this huge nit:

```suggestion
            // Tell WebRender to invalidate the caches for this image.
```

Reply:
It was a lot of work, but I've addressed this huge nit in [ec5c9df](https://github.com/servo/servo/pull/30390/commits/ec5c9df530355d1d406a497d9e4c9b538213f18f) :rofl: